internal/kubernetes/operator/features/crds.go

```diff
@@ -46,25 +49,40 @@ var (
 	ErrDocumentHasNoSchema       = errors.New("document contains no Schema")
 	ErrDocumentHasNoSpec         = errors.New("document contains no Spec")
 
-	versionsToResourcesMap = map[string][]string{
+	versionsToResourcesMap = map[string][]resource{
 		"2.0.0": {
-			ResourceAtlasDatabaseUser,
-			ResourceAtlasProject,
-			ResourceAtlasDeployment,
-			ResourceAtlasBackupSchedule,
-			ResourceAtlasBackupPolicy,
-			ResourceAtlasTeam,
-			ResourceAtlasDataFederation,
-			ResourceAtlasFederatedAuth,
+			resource{ResourceAtlasDatabaseUser, NopPatcher()},
+			resource{ResourceAtlasProject, NopPatcher()},
+			resource{ResourceAtlasDeployment, NopPatcher()},
+			resource{ResourceAtlasBackupSchedule, NopPatcher()},
+			resource{ResourceAtlasBackupPolicy, UnkownBackupPolicyFrequencyTypesPruner()},
+			resource{ResourceAtlasTeam, NopPatcher()},
+			resource{ResourceAtlasDataFederation, NopPatcher()},
+			resource{ResourceAtlasFederatedAuth, NopPatcher()},
 		},
 	}
 )
 
-func GetResourcesForVersion(version string) ([]string, bool) {
+type resource struct {
+	name    string
+	patcher Patcher
+}
+
+func majorVersion(version string) string {
```

Comment:
Major would be X.0.0 right? this is more like "first minor", or "unpatched version"

Reply:
this function was already there but inlined, this does not introduce new logic.